content/en/blog/2022/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,67 @@
+
+# Background
+
+The journey has taken us longer than anticipated and we continue to have work to do. Initially started with a design based
+on a reference design implementation from F5. The design lead to an [RFC](https://docs.google.com/document/d/1oT6pmRhOw7AtsldU0-HbfA0zA26j9LYiBD_eepeErsQ/edit?usp=sharing) that
+caused us to re-examine our approach. Notably, there were concerns about memory and performance issues that the community wanted
+to be addressed before implementation.
+
+## Experimental Dual Stack branch
+
+While the team was re-evaluating its options we created a new branch, [experimental-dual-stack](https://github.com/istio/istio/tree/experimental-dual-stack),
+to experiment and implement the reference design as we continued to learn. We will detail how to use and install artifacts
+from that branch in an upcoming blog, however, be aware that its initial intent was to learn how to approach development
+in a way that we can implement a big impacting feature within Istio without causing regressions. As such, it would be
+considered highly experimental to use--unit and integration tests do not currently pass--yet there are still people
+that are testing and using this in their environment.
+
+A big part of the original design was to duplicate a lot of the Envoy configuration. Notably, the listeners, clusters and routes
+with a new set of endpoints they can be referred to. The original design was created with customer requirements that
+specified client originating IPv4 requests should be proxied over IPv4, and the same for an originating IPv6 to be
+satisfied over IPv6 (we will refer to this as native IP family going forward).
+
+## Community Feedback
+
+Much of the feedback provided by the community for the original RFC was to change Envoy to better support dual-stack use cases
+internally instead of supporting this within Istio. This has led us to a [new design](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing)
+where we have taken lessons learned as well as feedback and have applied them.
+
+## Current Work
+
+We have worked with the Envoy community to address numerous concerns which is a reason why dual-stack enablement has
+taken us a while to implement. We have [matched IP Family for outbound listener](https://github.com/envoyproxy/envoy/issues/16804)
+and [supported multiple addresses per listener](https://github.com/envoyproxy/envoy/issues/11184). Alex Xu has also
+been working fervently to get long outstanding issues resolve, with the ability for Envoy to
+[bind to multiple addresses](https://github.com/envoyproxy/envoy/issues/9811) while we also work on a
+[smarter way to pick endpoints for dual-stack](https://github.com/envoyproxy/envoy/issues/21640). In total the team
+has submitted over a dozen PRs to Envoy and are working on a least a half dozen more to make Envoy adoption of
+dual stack easier for Istio.
```

Comment:
Besides the issues and PRs, I suggest that we can paste the API description here:
Multiple addresses support : https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto.html?highlight=additional_addresses
BindConfig support: https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-bindconfig